app/src/services/webtoon/webtoonService.js

```diff
@@ -2,12 +2,12 @@
 
 const webtoonRepository = require("../../repositories/webtoon/webtoonRepository");
 
-class webtoonService {
+class WebtoonService  {
     constructor(req) {
         this.body = this.body;
```

Comment:
생성자 내부에 this.body = this.body;로 작성되어 있는데 아마 this.body = req.body;를 의도하신 것 같습니다.
확인 후 수정해주시면 좋겠습니다!
```suggestion
this.body = req.body;
```

Reply:
수정하겠습니다!